lib/traject/config/sirsi_config.rb

```diff
@@ -1400,6 +1456,10 @@ def marc_008_date(byte6values, byte_range, u_replacement)
   accumulator << 'On order' if accumulator.empty?
   accumulator << 'Online' if context.output_hash['url_fulltext']
   accumulator << 'Online' if context.output_hash['url_sfx']
+  # There is similar logic in the SearchWorks HathiTrustLinks class
+  if context.output_hash['ht_access_sim']&.none? { |v| v == 'deny' || %w[allow:icus allow:pdus].include?(v) }
```

Comment:
Do we need to worry about `ht_access_sim` being empty (because `[].none? == true`)? Maybe it can't happen?

Reply:
It couldn't hurt!

Reply:
🤷‍♂️ ; I remember it being an issue over in searchworks, but, yeah, can't hurt.